index.bs

```diff
@@ -1131,14 +1178,14 @@ to filling the [=readable stream=]'s [=internal queue=] or changing its state. I
  The <dfn id="default-reader-closed" attribute for="ReadableStreamDefaultReader">closed</dfn>
```

Comment:
I would have liked to remove the `closed` getter and `cancel(reason)` methods on `ReadableStreamDefaultReader` and `ReadableStreamBYOBReader`, in favor for using the definitions from `ReadableStreamGenericReader`. But apparently that would break the links to MDN, so the CI doesn't let me do that here. 🤷

I could make the steps perform the steps for `ReadableStreamGenericReader.closed` and `.cancel(reason)`, but I'm not sure how to formalize that.

Reply:
We need to remove them; otherwise the spec is ill-formed. But, you can use `oldids=""` to make links go to the right place.

Reply:
> But, you can use `oldids=""` to make links go to the right place.

Thanks, that's *exactly* what I was looking for! 😄